Dockerfile

```diff
@@ -10,23 +10,27 @@ COPY vendor/php-parser/composer.lock /usr/src/app/vendor/php-parser/
 
 COPY package.json /usr/src/app/
 
-RUN curl --silent --location https://deb.nodesource.com/setup_5.x | bash - && \
-    apt-get update && apt-get install -y nodejs python python3 openssh-client php5-cli php5-json
+# not sure if this is ok to skip. apk's nodejs is 6.7.x:
+# RUN curl --silent --location https://deb.nodesource.com/setup_5.x | bash -
+RUN apk update && apk add nodejs python python3 php5-phar php5-openssl php5-cli php5-json php5-zlib
+
+# for bundler w/ flay fork... not necessary once off flay fork
+RUN apk add git
```

Comment:
Though it's a bit counterintuitive, it's better to collapse layers in this file when you are deleting things at the end. It helps keep image size down because the size is determined by the size at the end of the layer's command. In this case, I would collapse it to

```
RUN apk add git curl && \
    gem install bundler --no-ri --no-rdoc && \
    bundle install -j 4 && \
    curl -sS https://getcomposer.org/installer | php && \
    apk del --purge curl git
```